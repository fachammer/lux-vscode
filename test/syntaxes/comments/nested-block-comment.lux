#(This is a #(nested block) comment)