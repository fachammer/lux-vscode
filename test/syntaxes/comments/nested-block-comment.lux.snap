>#(This is a #(nested block) comment)
#^^ source.lux comment.block.lux
#  ^^^^^^^^^^ source.lux comment.block.lux
#            ^^ source.lux comment.block.lux comment.block.lux
#              ^^^^^^^^^^^^ source.lux comment.block.lux comment.block.lux
#                          ^ source.lux comment.block.lux comment.block.lux
#                           ^^^^^^^^ source.lux comment.block.lux
#                                   ^ source.lux comment.block.lux